lib/redcord/tasks/redis.rake

```diff
@@ -31,4 +32,18 @@ db_namespace = namespace :redis do
     migration_end = Process.clock_gettime(Process::CLOCK_MONOTONIC)
     puts "\nFinished in #{(migration_end - migration_start).round(3)} seconds"
   end
+
+  task :vacuum, [:model_name] => :environment do |t, args|
```

Comment:
can we vacuum all when no model is given?

Reply:
Yes - I can follow up with taking a splat argument so multiple model names can be passed in, and by default vacuum all models